sdk/src/store.rs

```diff
@@ -5533,2991 +5779,2928 @@ pub mod tests {
-                let (label, instance) = Claim::assertion_label_from_link(&hashed_uri.url());
-                claim
-                    .get_claim_assertion(&label, instance)
-                    .expect("Should find assertion");
-            }
-        }
-    }
-
-    /*  todo: disable until we can generate a valid file with no xmp
-    #[test]
-    fn test_manifest_no_xmp() {
-        let ap = fixture_path("CAICAI_NO_XMP.jpg");
-        assert!(Store::load_from_asset(&ap, true, None).is_ok());
-    }
-    */
-
-    #[test]
-    fn test_manifest_bad_sig() {
-        let ap = fixture_path("CE-sig-CA.jpg");
-        assert!(Store::load_from_asset(
-            &ap,
-            true,
-            &mut StatusTracker::with_error_behavior(ErrorBehavior::StopOnFirstError)
-        )
-        .is_err());
-    }
+            assert!(!report.logged_items().is_empty());
 
-    #[test]
-    fn test_unsupported_type_without_external_manifest() {
-        let ap = fixture_path("Purple Square.psd");
-        let mut report = StatusTracker::default();
-        let result = Store::load_from_asset(&ap, true, &mut report);
-        assert!(matches!(result, Err(Error::UnsupportedType)));
-        println!("Error report for {}: {:?}", ap.display(), report);
-        assert!(!report.logged_items().is_empty());
+            let errors: Vec<&LogItem> = report.filter_errors().collect();
+            assert!(errors[0].err_val.as_ref().unwrap().starts_with("IoError"));
+        }
 
-        assert!(report.has_error(Error::UnsupportedType));
-    }
+        #[test]
+        fn test_old_manifest() {
+            let ap = fixture_path("prerelease.jpg");
+            let mut report = StatusTracker::default();
+            let _r = Store::load_from_asset(&ap, true, &mut report);
 
-    #[test]
-    fn test_bad_jumbf() {
-        // test bad jumbf
-        let ap = fixture_path("prerelease.jpg");
-        let mut report = StatusTracker::default();
-        let _r = Store::load_from_asset(&ap, true, &mut report);
+            println!(
+                "Error report for {}: {:?}",
+                ap.display(),
+                report.logged_items()
+            );
 
-        // error report
-        println!("Error report for {}: {:?}", ap.display(), report);
-        assert!(!report.logged_items().is_empty());
+            assert!(!report.logged_items().is_empty());
 
-        assert!(report.has_error(Error::PrereleaseError));
-    }
+            let errors: Vec<&LogItem> = report.filter_errors().collect();
+            assert!(errors[0]
+                .err_val
+                .as_ref()
+                .unwrap()
+                .starts_with("Prerelease"));
+        }
 
-    #[test]
-    fn test_detect_byte_change() {
-        // test bad jumbf
-        let ap = fixture_path("XCA.jpg");
-        let mut report = StatusTracker::default();
-        Store::load_from_asset(&ap, true, &mut report).unwrap();
+        #[test]
+        #[ignore] // we no longer support these
```

Comment:
Should they be removed then?

Reply:
I just had to increase the indent on these tests. I didn't otherwise alter them.